packages/charts/examples/components/demo/chart-types/area/area-chart-bi-directional-stacked/area-chart-bi-directional-stacked-example.component.ts

```diff
@@ -61,31 +67,47 @@ export class AreaChartBiDirectionalStackedExampleComponent implements OnInit {
         // This example demonstrates a scenario with time on the X scale and a numeric value on the Y scale.
         const scalesTop: IXYScales = {
             x: xScale,
-            y: new LinearScale(),
+            y: new LinearScale().reverse(),
         };
 
         const scalesBottom: IXYScales = {
             x: xScale,
-            y: new LinearScale().reverse(),
+            y: new LinearScale(),
         };
 
         // Here we assemble the complete chart series.
-        const seriesSetTop: Partial<IChartSeries<IAreaAccessors>>[] = getData().map(d => ({
+        const seriesSetTop: IChartSeries<IAreaAccessors>[] = getDataTop().map(d => ({
             ...d,
             renderer,
             accessors,
             scales: scalesTop,
         }));
-        this.chartAssistTop.update(seriesSetTop as IChartSeries<IAreaAccessors>[]);
 
-        const seriesSetBottom: Partial<IChartSeries<IAreaAccessors>>[] = getData().map(d => ({
+        const seriesSetBottom: IChartSeries<IAreaAccessors>[] = getDataBottom().map(d => ({
             ...d,
             renderer,
             accessors,
             scales: scalesBottom,
         }));
+
+        scalesTop.y.domainCalculator = scalesTop.x.domainCalculator =
+            domainWithAuxiliarySeries(() => seriesSetBottom, getAutomaticDomain);
+        scalesBottom.y.domainCalculator = scalesBottom.x.domainCalculator =
+            domainWithAuxiliarySeries(() => seriesSetTop, getAutomaticDomain);
+
+        this.chartAssistTop.update(seriesSetTop as IChartSeries<IAreaAccessors>[]);
         this.chartAssistBottom.update(seriesSetBottom as IChartSeries<IAreaAccessors>[]);
     }
+
+    public onSelectedChange(legendSeries: IChartAssistSeries<any>, value: boolean, currentChartAssist: ChartAssist) {
```

Comment:
Maybe `selected` for better clarity?

```suggestion
    public onSelectedChange(legendSeries: IChartAssistSeries<any>, selected: boolean, currentChartAssist: ChartAssist) {
```